### PR TITLE
QA: adjust Y axis spaces of Work Gallery items

### PR DESCRIPTION
**Pull request by nahbee10**
### Description

- reduce the space between bottom of image and top of caption(.625rem -> .325rem) to make the caption more connected to its image
- reduce the space between bottom of caption and top of the image(1rem-> .9rem) to make the space look even to R/L gutter of the website visually

### Type(s) of changes

- [ ] Bug fix
- [ ] New feature
- [x] Update to an existing feature

### Motivation for PR

QA item from Notion([LINK](https://www.notion.so/garden3d/On-mobile-the-vertical-padding-between-modules-should-appear-the-same-as-our-standard-gutters-to-th-4bf9754631f3420f94e0fcacc81ebe14?pvs=4))

### How Has This Been Tested?

Tested with [preview link](https://sanctu-dot-eb7xu2jnx-sanctucompu.vercel.app/) on mobile browser

### Applicable screenshots

- before:
Screen Shot 2023-02-10 at 11 38 15 AM
Screen Shot 2023-02-10 at 11 38 04 AM

- after:
Screen Shot 2023-02-10 at 11 37 25 AM
Screen Shot 2023-02-10 at 11 37 13 AM

### CODE REVIEW

File: components/WorkSectionAsGallery.js [hunk 1]
```diff
@@ -45,12 +45,12 @@ const WorkGalleryAssetBlock = ({ assetBlock }) => {
...
         <>
           <div className="flex col-8 pb0 md:pb1 items-end md:flex-row flex-col">
-            <div className="block md:col-4 col-8">
+            <div className="block md:col-4 col-8 md:mb0 mb_9">
               <div className="relative">
                 <Work work={works[0]} width="50vw" />
               </div>
             </div>
-            <div className="block md:col-4 col-8 md:ml_5 md:ml1">
+            <div className="block md:col-4 col-8 md:ml_5 md:ml1 md:mb0 mb_9">
```
Comment (mokaymokay) [inline components/WorkSectionAsGallery.js:53]: can we fix the conflicting classes `md:ml_5 md:ml1` here?

File: styles/components/Markdown.scss [hunk 2]
```diff
@@ -9,6 +9,9 @@
     @include media('xl') {
       margin-bottom: $p-xl-line-height;
     }
+    &:last-child {
```
Comment (mokaymokay) [inline styles/components/Markdown.scss:12]: nitpick: can we add an empty line between declarations plz

**Review by mokaymokay — APPROVED** [review]
This is looking soo good, thank you for the attention to detail!!! Just one last thing, can we please add in-line documentation to `Work` and `WorkSectionAsGallery` so we remember why we are choosing these values? <3

File: components/WorkSectionAsGallery.js (hunk 1 above)
Comment (nahbee10) [reply components/WorkSectionAsGallery.js:53]: yes! removed the first one!!

**Pull request merged**